src/appleseed/renderer/modeling/object/diskobject.h

```diff
@@ -34,6 +34,7 @@
 
 // appleseed.foundation headers.
 #include "foundation/math/vector.h"
+#include "foundation/math/ray.h"
```

Comment:
Order alphabetically.

But actually, if we use `ShadingRay` instead of `foundation::Ray3d` this header can go.